website/styles.css

```diff
@@ -21,3 +21,7 @@ h3 {
   width: 460px;
   margin: 40px;
 }
+
+.wide {
+  width: 1000px;
+}
```

Comment:
If the stress test looks weird at 460px, rather than introduce another size of example, can we increase the default width, maybe to ~550px?

Reply:
Yeah it seems squished horizontally but that's going to be the case with any large data set. I changed the width to 550px though